### PR TITLE
Fixed some issues in the cluster api endpoint.

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard_v2/models/nodb.py [hunk 1]
```diff
@@ -821,7 +822,10 @@ def handle_field(field):
         return model_args
 
     def __init__(self, *args, **kwargs):
-        self._state = mock.Mock()
```
Comment (rjfd) [inline src/pybind/mgr/dashboard_v2/models/nodb.py:824]: You must also remove the `from mock import mock` line in the beginning of the file.

**Review by rjfd — CHANGES_REQUESTED** [review]
Also, please add the the `mock` import in `test_cluster.py`.

File: src/pybind/mgr/dashboard_v2/models/nodb.py (hunk 1 above)
Comment (sebastian-philipp) [reply src/pybind/mgr/dashboard_v2/models/nodb.py:824]: fixed

**Review by p-se — APPROVED** [review]

File: src/pybind/mgr/dashboard_v2/module.py [hunk 2]
```diff
@@ -94,14 +94,26 @@ def handle_command(self, cmd):
...
                 setattr(Module.ApiRoot, ctrl._cp_path_, ins)
 
+        @cherrypy.expose
+        def index(self):
+            tpl = """API Endpoints:<br>
+            <ul>
+            {lis}
+            </ul>
+            """
+            lis = ['<li><a href="{}">{}</a></li>'.format(ctrl._cp_path_, ctrl.__name__)
```
Comment (votdev) [inline src/pybind/mgr/dashboard_v2/module.py]: I suggest to rename the variable ``lis`` to ``endpoints``.

**Review by votdev — APPROVED** [review]

File: src/pybind/mgr/dashboard_v2/module.py (hunk 2 above)
Comment (sebastian-philipp) [reply src/pybind/mgr/dashboard_v2/module.py]: fixed

**Review by rjfd — APPROVED** [review]
lgtm

**Review by s0nea — APPROVED** [review]

**Pull request merged**